CrazyCanvas/Include/Match/MatchServer.h

```diff
@@ -11,6 +11,8 @@
 
 #include "Lobby/Player.h"
 
+typedef std::pair<LambdaEngine::Entity, float32> PlayerRespawnTimers;
```

Comment:
```suggestion
typedef std::pair<LambdaEngine::Entity, float32> PlayerRespawnTimer;
```

Reply:
I suppose it is a single player per pair?

Reply:
That is correct! ^^ I forgot to remove the 's' when I removed the TArray from the typedef ^^